scheduler/src/main/java/io/camunda/zeebe/scheduler/future/CompletableActorFuture.java

```diff
@@ -231,6 +252,7 @@ public Throwable getException() {
 
   private void notifyBlockedTasks() {
```

Comment:
:upside_down_face:  to improve naming: `notifyAllBlocked` which calls `notifyBlockedTasks` and `notifyBlockedCallbacks`?